client/components/inventory.jsx

```diff
@@ -62,3 +103,63 @@ function Article(props) {
     </div>
   );
 }
+
+function Articles(props) {
+  const { articles, articleType } = props.inventoryState;
+  const numPlaceholders = [];
+  let placeholderType;
+  if (articleType === 'articles') {
+    placeholderType = 'hoodie';
+  } else {
+    placeholderType = articleType;
+  }
+  if (articles.length % 2 === 0) {
+    for (let i = 0; i < 8; i++) {
+      numPlaceholders.push('placeholder');
+    }
+  } else {
+    for (let i = 0; i < 7; i++) {
+      numPlaceholders.push('placeholder');
+    }
+  }
```

Comment:
I still can't quite figure out what these `for` loops are doing. like, why do you need so many placeholders? I sort of get making it an even number, but do we need another 7 if we already have 23? 😂